CONTRIBUTING.md

```diff
@@ -155,6 +162,62 @@ git push origin HEAD # This will close the PR
 * Keeps commit history clean and easy to read
 
 
+## App Contribution Guidelines
```

Comment:
Wondering out loud if we should change the title of this section to "Frontend Contribution Guidelines" for extra clarity. There's some ambiguity about whether a web3 "app" means just the frontend or frontend + contracts.

Reply:
Should we rename the root folder from `app` to `frontend` then? I went with app here so they'd be consistent

Reply:
I think it's ok to leave the folder named `app`, but for someone not yet familiar with that naming convention, using Frontend here might be clearer. Don't feel strongly about it, happy to hear what others think and fine either way.

Reply:
Updated to frontend in 42a8e96d95f2138fea0243f7438bb8c5d7f4f6a6